docs/3.0.0-beta.x/community/contribute-with-code.md

```diff
@@ -0,0 +1,103 @@
+# Contribute with Code
+
+**NOTE:** All efforts to contribute are highly appreciated, we recommend you talk to a maintainer **before** to spending much time making a pull request that may not align with the project roadmap.
+
+## Before Submitting a Pull Request
+
+**Before submitting your pull request**, fulfill the following requirements:
+
+- Fork the repository and create your branch from `master`.
+- Run `yarn setup` in the repository root.
+- If you’ve fixed a bug or added code that should be tested, add the tests, and then link the corresponding issue in either your commit or your PR!
+- Ensure the test suites are passing:
+  - `yarn test:unit`
+  - `yarn test:front`
+- Use a linter on your code (`yarn lint`).
+- When you write the Pull Request, please follow the [Pull Request Template](https://github.com/strapi/strapi/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
+
+## Technical Requirements
+
+- You have [Node](https://nodejs.org/en/) at v10.x.x only and [Yarn](https://yarnpkg.com/en/) at v1.2.0+.
```

Comment:
Should state Node LTS instead of Node 10 as v12 goes LTS in october

Reply:
Clarified this in the three locations needed.  Thank you. @derrickmehaffy